Makefile

```diff
@@ -21,7 +21,9 @@ ARCH ?= $(shell uname -m)
 
 ifeq ($(ARCH),aarch64)
   ARCH = arm64
-else ($(ARCH),x86_64)
+else
```

Comment:
Following `else ifeq` style is much clear, but according to [stackflow](https://stackoverflow.com/questions/34761685/extraneous-text-after-else-directive), this style does not work on Windows. Hence I changed safer way.

```
ifeq ($(ARCH),aarch64)
  ARCH = arm64
else ifeq ($(ARCH),x86_64)
  ARCH = amd64
endif
```

Reply:
Thanks

Reply:
Thanks! I have missed that "else ifeq" , sorry for that